foreman/data_refinery_foreman/foreman/management/commands/test_rerun_salmon_old_samples.py

```diff
@@ -0,0 +1,288 @@
+
+        kv = ComputationalResultAnnotation()
+        kv.data = {"index_length": "short"}
+        kv.result = quant_result
+        kv.is_public = True
+        kv.save()
+
+        # In prod the filename pattern will involve the timestamp
+        # but here we're using the accession code so we can find
+        # the archive file for the current sample.
+        archive_filename = "result-" + accession_code + ".tar.gz"
+        archive_file = ComputedFile()
+        archive_file.filename = archive_filename
+        archive_file.absolute_file_path = os.path.join(experiment_dir, archive_filename)
+        archive_file.is_public = False
+        archive_file.is_smashable = False
+        archive_file.is_qc = False
+        archive_file.result = quant_result
+        archive_file.size_in_bytes = 12345
+        archive_file.save()
+
+        quant_file = ComputedFile()
+        quant_file.filename = "quant.sf"
+        quant_file.absolute_file_path = experiment_dir + "/quant_files/" + accession_code + "_output/quant.sf"
+        quant_file.is_public = False
+        quant_file.is_smashable = False
+        quant_file.is_qc = False
+        quant_file.result = quant_result
+        quant_file.size_in_bytes = 12345
+        quant_file.s3_bucket = "bucket"
+        quant_file.s3_key = "key"
+        quant_file.save()
+
+        SampleResultAssociation.objects.get_or_create(
+            sample=sample,
+            result=quant_result
+        )
+
+    # Create another OrganismIndex with a newer version of
+    transcriptome_processor = Processor()
+    transcriptome_processor.name = "Transcriptome"
+    transcriptome_processor.version = "v9.9.10"
+    transcriptome_processor.docker_image = "dr_transcriptome"
+    transcriptome_processor.environment = '{"some": "environment"}'
+    transcriptome_processor.save()
+    computational_result_short = ComputationalResult(processor=transcriptome_processor)
+    computational_result_short.save()
+
+    organism_index = OrganismIndex()
+    organism_index.index_type = "TRANSCRIPTOME_SHORT"
+    organism_index.organism = zebrafish
+    organism_index.result = computational_result_short
+    organism_index.absolute_directory_path = "/home/user/data_store/ZEBRAFISH_INDEX/SHORT"
+    organism_index.salmon_version='v0.13.0' # DIFFERENT SALMON VERSION
+    organism_index.save()
+
+    comp_file = ComputedFile()
+    # This path will not be used because we already have the files extracted.
+    comp_file.absolute_file_path = "/home/user/data_store/ZEBRAFISH_INDEX/SHORT/zebrafish_short.tar.gz"
+    comp_file.result = computational_result_short
+    comp_file.size_in_bytes=1337
+    comp_file.sha1="ABC"
+    comp_file.s3_key = "key"
+    comp_file.s3_bucket = "bucket"
+    comp_file.save()
+
+
+    # Create the already processed samples along with their
+    # ComputationalResults and ComputedFiles. They don't need
+    # original files for this test because we aren't going to run
+    # salmon quant on them.
```

Comment:
I think you copied this comment from the other test because you do create original files here.